modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/raft/RocksDbClusterStateStorage.java

```diff
@@ -86,12 +87,17 @@ public RocksDbClusterStateStorage(Path dbPath, String nodeName) {
 
     @Override
     public CompletableFuture<Void> start() {
-        init();
+        // Delete existing data, relying on log playback.
+        try {
+            RocksDB.destroyDB(dbPath.toString(), options);
+        } catch (RocksDBException e) {
+            return failedFuture(new CmgStorageException("Unable to drop RocksDb storage", e));
+        }
 
-        return nullCompletedFuture();
+        return init();
```

Comment:
Please be careful, this method is used in two places and in the second an exception must be thrown.